ReactWindows/ReactNative.Shared/UIManager/ViewProps.cs

```diff
@@ -85,6 +85,10 @@ public static class ViewProps
         public const string TextDecorationLine = "textDecorationLine";
         public const string AllowFontScaling = "allowFontScaling";
 
+        public const string AccessibilityTraits = "accessibilityTraits";
```

Comment:
If you're adding these consts here, can we also use them in the `[ReactProp]` declarations?

Reply:
Done, + added one more property into the mix

---
In reply to: [189323869](https://github.com/Microsoft/react-native-windows/pull/1839#discussion_r189323869) [](ancestors = 189323869)